config/polymoprh_rules.txt

```diff
@@ -0,0 +1 @@
+<span class='danger'>Note that you are allowed to act as an antagonist while transformed into a hostile mob, unless you volunteered for or seeked out transformation.</span>
```

Comment:
Filename is misspelled. Code looks for polymorph_rules.txt, not polymoprh_rules.txt